BrainPortal/lib/cbrain_task_generators/schema_task_generator.rb

```diff
@@ -166,6 +165,20 @@ def integrate(register: true, create_tool_config: false, multi_version: false)
         })[partial]
       end
 
+      # Write out a help file for this Boutiques task
+      helpFileName = name + "_help.html" 
```

Comment:
I believe we talked about this before, but is there no way use them in memory? For instance, in _tools_table.html.erb, using something other than overlay_ajax_link for Boutiques tools? 

If it was a rake task, I suppose it would be an addition to the "plugins:install" command?